src/classes/applicant.py

```diff
@@ -0,0 +1,86 @@
+from uuid import UUID, uuid4
+from datetime import date
+from user import User
+
+
+class Applicant(User):
+
+    def __init__(
+            self,
+            first_name: str,
+            last_name: str,
+            date_of_birth: date,
+            street: str,
+            postal_code: int,
+            city: str,
+            email: str,
+            phone_number: str,
+            rating: float = 0.0
```

Comment:
Rating doesn't get passed into the class when created so it should be removed here